### PR TITLE
Page chart width

### PR DESCRIPTION
**Pull request by LilMDShin**

### CODE REVIEW

File: src/app/design/components/popover-menu/popover-menu.component.scss [hunk 1]
```diff
@@ -0,0 +1,95 @@
+.popover-menu--container {
```
Comment (vincent-inviseo) [inline src/app/design/components/popover-menu/popover-menu.component.scss]: Have you check name selector in cscc and TS files ? In scss selector name is fcs-popover.... And html app-popver

File: src/app/pyscada-lite/graphs/doughnut/doughnut.component.ts [hunk 2]
```diff
@@ -63,15 +72,50 @@ export class DoughnutComponent implements AfterViewInit, OnChanges {
...
+          this.chart.datas.variables = variablesValues;
+          this.updateDoughnut(this.chart);
+        });
+      }
+    }
+  }
+
+  /*
+
+  public setValuesDataset(chart: any) {
```
Comment (vincent-inviseo) [inline src/app/pyscada-lite/graphs/doughnut/doughnut.component.ts:100]: Comments

File: src/app/pyscada-lite/graphs/doughnut/doughnut.component.ts [hunk 3]
```diff
@@ -88,7 +132,7 @@ export class DoughnutComponent implements AfterViewInit, OnChanges {
         ctx.fillStyle = 'black';
         ctx.textAlign = 'center';
         ctx.textBaseline = 'middle';
-        ctx.fillText((100 * value / this.sumValues).toFixed(2) + ' %', left + width / 2, top + height / 2);
+        // ctx.fillText((100 * value / this.sumValues).toFixed(2) + ' %', left + width / 2, top + height / 2);
```
Comment (vincent-inviseo) [inline src/app/pyscada-lite/graphs/doughnut/doughnut.component.ts:133]: Comment is utils ?

**Review by vincent-inviseo — CHANGES_REQUESTED** [review]

File: src/app/pyscada-lite/graphs/doughnut/doughnut.component.ts (hunk 3 above)
Comment (LilMDShin) [reply src/app/pyscada-lite/graphs/doughnut/doughnut.component.ts:133]: Handles percentage at center of doughnut, needs real data to adapt

**Review by vincent-inviseo — APPROVED** [review]

**Pull request merged**